README.md

```diff
@@ -21,14 +21,14 @@ So first off: What is iceoryx?
 iceoryx is an inter-process-communication (IPC) middleware for various operating systems (currently we support Linux, MacOS and QNX).
 It has its origins in the automotive industry, where large amounts of data have to be transferred between different processes
 when it comes to driver assistance or automated driving systems. However, the efficient communication mechanisms can also be applied
-to a wider range of use cases, e.g. in the field of robotics or game development. 
+to a wider range of use cases, e.g. in the field of robotics or game development.
 
 <p align="center">
 <img src="https://user-images.githubusercontent.com/8661268/74612998-b962bc80-510a-11ea-97f0-62f41c5d287b.gif" width="100%">
 </p>
 
-iceoryx uses a true zero-copy, shared memory approach that allows to transfer data from publishers to subscribers without a single copy. 
-This ensures data transmissions with constant latency, regardless of the size of the payload. For more information have a look at the 
+iceoryx uses a true zero-copy, shared memory approach that allows to transfer data from publishers to subscribers without a single copy.
```

Comment:
:heart: 

Where is my markdown linter CI job? :smirk:

Reply:
@mossmaurice That sounds like a request in a ticket, please create one and give some info about what you would expect what the job should do and additional requirements (custom or standard rules? line  length? Exceptions from Linting etc.)

Reply:
@dkroenke Done https://github.com/eclipse-iceoryx/iceoryx/issues/921